addons/point_of_sale/static/src/app/screens/product_screen/control_buttons/control_buttons.xml

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates id="template" xml:space="preserve">
     <t t-name="point_of_sale.ControlButtons">
-        <div t-attf-class="flex-{{!props.wrapped ? (ui.isSmall ? 'column' : 'wrap') : 'wrap'}}" class="control-buttons d-flex bg-300 border-bottom">
+        <div t-attf-class="flex-{{!props.wrapped ? (ui.isSmall ? 'column' : 'wrap gap-2') : 'wrap bg-300'}}" class="control-buttons d-flex">
             <!-- All buttons always displayed -->
             <SelectPartnerButton partner="partner" t-if="props.wrapped"/>
             <t t-if="props.wrapped || (ui.isSmall and !props.wrapped)">
                 <button t-if="pos.models['account.fiscal.position'].length"
-                    class="control-button o_fiscal_position_button btn btn-light rounded-0 fw-bolder" t-on-click="() => this.clickFiscalPosition()">
+                    class="control-button o_fiscal_position_button fw-bolder"
+                    t-attf-class="btn {{ ui.isSmall ? 'btn-secondary rounded' : 'btn-light rounded-0' }}"
```

Comment:
could you find a way to not duplicate this line? Perhaps have a function in the js part that returns the button classes and use that everywhere